about/policy/sgid/index.md

```diff
@@ -10,16 +10,33 @@ categories: []
 date: 2019-12-04 10:01:27 -0500
 ---
 
-The following applies to data in the internal SGID database.
+The following policies apply to data in the internal SGID geodatabase, which is a curated database of state-wide datasets. It is maintained as an internal, AGRC-only Esri Enterprise Geodatabase and is available to everyone as the cloud-hosted, PostGIS-backed [OpenSGID]({% link sgid/open-sgid/index.md %}). The contents of the SGID are also uploaded to the Utah SGID groups of our [ArcGIS Online Organization](https://utah.maps.arcgis.com).
```

Comment:
**thought**: I think of the SGID as the open sgid, open data, and gis website. I don't think we need to describe the open sgid as a cloud hosted yada yada as that is an implementation detail. I don't think we really promote arcgis online as open data is the "curated" list of agol items.

```suggestion
The following policies apply to data in the internal SGID geodatabase, which is a curated database of state-wide datasets. It is maintained as an internal, AGRC-only Esri Enterprise Geodatabase and is available to everyone as the [OpenSGID]({% link sgid/open-sgid/index.md %}) and [Open Data](https://opendata.gis.utah.gov).
```

Reply:
I guess I wanted to toot the open source horn of the OpenSGID, but you're right, it's cleaner without it.

I'm working under the idea that this policy refers primarily to things in the internal SGID. Because AGOL and Open Data are both supersets of the internal SGID, I'm trying to narrow that last sentence down to just refer to the Utah SGID groups in AGOL... though now that I think about it, that distinction breaks down as soon as someone shares their external data to one of those groups to get it in Open Data. 

Perhaps we change the last line to "available to everyone as the OpenSGID and as part of our Open Data site"?

Reply:
let's try that. I updated the suggestion.